### PR TITLE
Feature/add more 37 features

### PR DESCRIPTION
**Pull request by jsteemann**
Added documentation about 3.7 features and changes

### CODE REVIEW

File: 3.7/release-notes-new-features37.md [hunk 1]
```diff
@@ -445,6 +473,64 @@ FOR doc IN collection1
...
+server has gone down and rejoins the cluster.
+
+The previous protocol operated in three passes. The first took time proportional 
+to the total number of documents in the shard, while the second and third passes 
+were linear in the number of documents which changed and the size of the changed 
+documents, respectively. 
+
+The new protocol introduced in ArangoDB 3.7 makes the first pass _constant_ with 
+respect to the size of the shard and the differences between leader and follower
+shard, so it is longer linear in the total number of documents in the shard.
```
Comment (Simran-B) [inline 3.7/release-notes-new-features37.md:520]: Is this what you meant to say?
```suggestion
shard, so it is no longer linear to the total number of documents in the shard.
```

File: 3.7/release-notes-new-features37.md [hunk 2]
```diff
@@ -651,14 +746,91 @@ Web UI
 
 The interactive description of ArangoDB's HTTP API (Swagger UI) shows the
 endpoint and model entries collapsed by default now for a better overview.
+ 
+The bundled version of Swagger has been upgraded to 3.25.1. This change has
+also been backported to ArangoDB v3.6.5.
```
Comment (Simran-B) [inline 3.7/release-notes-new-features37.md:751]:
```suggestion
also been backported to ArangoDB v3.6.4.
```

File: 3.7/release-notes-new-features37.md [hunk 3]
```diff
@@ -651,14 +746,91 @@ Web UI
...
 
 Metrics
 -------
 
-The amount of exported metrics has been extended and is now available in a
-format compatible with Prometheus. You can now easily scrape on `/_admin/metrics`.
+The amount of exported metrics for monitoring has been extended and is now 
+available in a format compatible with Prometheus. You can now easily scrape 
+on `/_admin/metrics`.
 See [here](http/administration-and-monitoring-metrics.html).
```
Comment (Simran-B) [inline 3.7/release-notes-new-features37.md:759]:
```suggestion
See [Metrics HTTP API](http/administration-and-monitoring-metrics.html).
```

File: 3.7/release-notes-new-features37.md [hunk 4]
```diff
@@ -680,10 +852,31 @@ This change simplifies the installation procedures and internal code paths.
...
+
+The OpenLDAP version used for the LDAP integration in the ArangoDB Enterprise 
+Edition has been upgraded to 2.4.50.
+This change has been backported to ArangoDB v3.6.5 as well.
+
+### Added libunwind library dependency
+
+The Linux builds of ArangoDB now use the third-party library
+[libunwind](https://github.com/libunwind/libunwind){:target="_blank"} to get
+backtraces and symbolize stack frames.
```
Comment (Simran-B) [inline 3.7/release-notes-new-features37.md:869]: ?
```suggestion
backtraces and symbolized stack frames.
```

File: 3.7/release-notes-new-features37.md [hunk 5]
```diff
@@ -230,11 +231,15 @@ batching of requests.
...
-single call, so all queries with an offset or the fullCount option enabled will
-benefit from this change straight away. This also holds true for subqueries,
+single call, so all queries with a LIMIT offset or the fullCount option enabled 
+will benefit from this change straight away. This also holds true for subqueries,
 hence the existing AQL optimizer rule `splice-subqueries` is now able to
 optimize all subqueries and is enabled by default.
 
+The query planner can now also reuse internal registers that were allocated for 
+storing temporary results inside subqueries, but not outside of subqueries.
+which are used inside subqueries only.
```
Comment (Simran-B) [inline 3.7/release-notes-new-features37.md:241]:
```suggestion
```

File: 3.7/release-notes-new-features37.md (hunk 4 above)
Comment (jsteemann) [reply 3.7/release-notes-new-features37.md:868]: No, it should be "symbolize" here, as it should be a verb and not an adjective.
I will add an extra "to" here to make it more clear.

File: 3.7/release-notes-new-features37.md [hunk 6]
```diff
@@ -680,10 +851,31 @@ This change simplifies the installation procedures and internal code paths.
...
+
+The OpenLDAP version used for the LDAP integration in the ArangoDB Enterprise 
+Edition has been upgraded to 2.4.50.
+This change has been backported to ArangoDB v3.6.5 as well.
+
+### Added libunwind library dependency
+
+The Linux builds of ArangoDB now use the third-party library
+[libunwind](https://github.com/libunwind/libunwind){:target="_blank"} to get
+backtraces and symbolize stack frames.
```
Comment (jsteemann) [inline 3.7/release-notes-new-features37.md:868]:
```suggestion
backtraces and to symbolize stack frames.
```

File: 3.7/release-notes-new-features37.md [hunk 7]
```diff
@@ -680,10 +851,31 @@ This change simplifies the installation procedures and internal code paths.
...
+Edition has been upgraded to 2.4.50.
+This change has been backported to ArangoDB v3.6.5 as well.
+
+### Added libunwind library dependency
+
+The Linux builds of ArangoDB now use the third-party library
+[libunwind](https://github.com/libunwind/libunwind){:target="_blank"} to get
+backtraces and to symbolize stack frames.
+
+Building against libunwind can be turned off at compile time using the
```
Comment (jsteemann) [inline 3.7/release-notes-new-features37.md:870]:
```suggestion
Building with libunwind can be turned off at compile time using the
```

**Pull request merged**